src/Network/Pusher/Base.hs

```diff
@@ -19,6 +20,23 @@ data ChannelInfo = ChannelInfo { occupied :: Bool
                                , subscriptionCount :: Maybe Int
                                } deriving Show
 
+
+data PartialChannel = PartialChannel { pUserCount :: Maybe Int } deriving Show
+
+instance FromJSON PartialChannel where
+  parseJSON (Object v) = PartialChannel <$> (v .:? "user_count")
+  parseJSON _ = mzero
+
+data Channel = Channel { name :: String, cUserCount :: Maybe Int } deriving Show
```

Comment:
It seems odd that everything else is prefixed, but `name` here is not. Is there a particular reason for this?